src/librados/librados.cc

```diff
@@ -266,7 +266,8 @@ void librados::ObjectReadOperation::omap_get_vals(
   int *prval)
 {
   ::ObjectOperation *o = &impl->o;
-  o->omap_get_vals(start_after, filter_prefix, max_return, out_vals, prval);
```

Comment:
looks like these librados test changes slipped into the wrong commit - they wouldn't build yet here